Makefile

```diff
@@ -42,3 +42,12 @@ build-sdk:
 # usage: make check-version VERSION=1.0.0
 check-version:
 	sh ./build/version.sh $(VERSION)
+
+all:
+	@echo "############################### Make all ###############################"
```

Comment:
Can you use the debug statements to respective targets and use make target dependency instead? 
`all: install-bsc archive`